pkg/export/export.go

```diff
@@ -0,0 +1,31 @@
+package export
+
+import (
+	"regexp"
+
+	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/yaml"
+)
+
+// pipelineExportRemoveExtraFields some of the fields are not getting skipped, when set as nil,
+// like creationTimeStamp since it's a time.Time that is not a pointer yaml would display a creationTimeStamp: null
+// let's do some regexp magic here, which should hopefully safe.
+func pipelineExportRemoveExtraFields(input string) string {
+	m1 := regexp.MustCompile(`\n  creationTimestamp: null\n`)
```

Comment:
`\n  creationTimestamp: null\n`
Do we need two spaces in the regex

Reply:
Yep that's how it matches ,what do you suggest we do ? Having a \s* ?

Reply:
My editor showed an error for this :smile_cat: just wanted to check.
/lgtm